data/json/items/comestibles.json

```diff
@@ -14654,11 +14654,36 @@
         "tool" : "null",
         "volume" : 1,
         "charges" : 3,
-        "bashing" : 1,
         "fun" : 4
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "fresh_fries",
+        "name" : "French fries",
+        "name_plural" : "French fries",
+        "weight" : 115,
+        "color" : "yellow",
+        "addiction_type" : "none",
+        "spoils_in" : 72,
+        "stim" : 0,
+        "container" : "box_small",
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : -1,
+        "healthy" : -1,
+        "addiction_potential" : 0,
+        "nutrition" : 3,
+        "description" : "Deep fried potatoes with a touch of salt.  Crunchy and delicious.",
+        "price" : 120,
+        "material" : "junk",
+        "tool" : "null",
+        "volume" : 1,
+        "charges" : 3,
+        "flags" : ["EATEN_HOT"],
+        "fun" : 3
```

Comment:
I added in the eaten hot bonus so that they would have a good bonus when fresh but would have a lover one once they are no longer hot. Cold fries aren't very good.